CMakeLists.txt

```diff
@@ -95,16 +95,24 @@ endif()
 if (SKIP_PYBIND11)
   message(STATUS "SKIP_PYBIND11 set - disabling python bindings")
 else()
-  include(IgnPython)
-  find_package(PythonLibs QUIET)
-  if (NOT PYTHONLIBS_FOUND)
-    IGN_BUILD_WARNING("Python is missing: Python interfaces are disabled.")
-    message (STATUS "Searching for Python - not found.")
+  #include(IgnPython) TODO: allow to specify for what it should search and then
+  # the code below can be removed; e.g. pybind needs Interpreter and Development components
+  # see https://pybind11.readthedocs.io/en/stable/cmake/index.html#new-findpython-mode
+  if(${CMAKE_VERSION} VERSION_LESS "3.12.0")
+    find_package(PythonLibs QUIET)
+    set(Python3_FOUND ${PYTHONLIBS_FOUND})
```

Comment:
I think we also need the following from `IgnPython` to get `Python3_EXECUTABLE`, which is used in the python binding tests.

```
  find_package(PythonInterp ${IGN_PYTHON_VERSION} QUIET)

  if(PYTHONINTERP_FOUND)
    set(Python3_Interpreter_FOUND ${PYTHONINTERP_FOUND})
    set(Python3_EXECUTABLE ${PYTHON_EXECUTABLE})
  endif()
```

Reply:
That's true and I just pushed a fix for it.